internal/controllers/reconciliation/controller.go

```diff
@@ -124,13 +117,7 @@ func (c *Controller) Reconcile(ctx context.Context, req resource.Request) (ctrl.
 	// Find the current and (optionally) previous desired states in the cache
 	var prev *resource.Resource
 	resource, visible, exists := c.resourceClient.Get(ctx, comp.Status.GetCurrentSynthesisUUID(), req.Resource)
-	if !exists {
-		// Returning is safe because filling the cache will enqueue a new work item
-		logger.V(1).Info("dropping work item because the corresponding synthesis no longer exists in the cache")
-		return ctrl.Result{}, nil
-	}
-	if !visible {
-		logger.V(1).Info("skipping because the resource isn't visible yet")
```

Comment:
These logs are too chirpy and I've personally never found them useful. Open to keeping them but I tend to favor less log noise